src/models/members.model.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const memberSchema = new Schema({
+  Name: {
+    type: String,
+    required: true,
+  },
+  DateOfBirth: {
```

Comment:
thường các tên truờng sẽ viết thường nhé, đặt tên theo chuẩn snake_case
```suggestion
  date_of_birth
```